src/mission.h

```diff
@@ -55,6 +57,7 @@ enum mission_goal {
     MGOAL_ASSASSINATE,       // Kill a given NPC
     MGOAL_KILL_MONSTER,      // Kill a particular hostile monster
     MGOAL_KILL_MONSTER_TYPE, // Kill a number of a given monster type
+    MGOAL_KILL_MONSTER_SPEC, // Kill a number of monsters from a given species
```

Comment:
I'm not sure this insert will not affect the goals below this line since enumeration will be shifted. It's much safer to add a new value right before NUM_GOAL.